metron-interface/metron-alerts-host/src/main/java/org/apache/metron/ui/AlertsApplication.java

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
Our formatting for Java is generally 2 spaces for indents.  I don't feel like it's much effort and might as well make it consistent.

Reply:
Done